tests/songController.test.ts

```diff
@@ -0,0 +1,38 @@
+import request from "supertest";
+import { newApp } from "../src/app";
+import fs from "fs";
+
+const app = newApp();
+
+describe("Upload a pdf", () => {
```

Comment:
Remember to wrap async functions in async functions, else the app may stop executing before the inside `async` functions complete.

```suggestion
describe("Upload a pdf", async () => {
```

Reply:
I did not know that! Thank you!